tex/generic/pgf/graphdrawing/lua/pgf/gd/control/ComponentDistance.lua

```diff
@@ -23,6 +23,10 @@ local declare       = require "pgf.gd.interface.InterfaceToAlgorithms".declare
 --
 -- @end
 
+\begin{codeexample}[setup code,hidden]
+    \usetikzlibrary{graphs,graphdrawing}
+    \usegdlibrary{trees}
+\end{codeexample}
```

Comment:
Comment this out, otherwise it is invalid Lua code, e.g.
```lua
--[[
\begin{codeexample}[setup code,hidden]
    \usetikzlibrary{graphs,graphdrawing}
    \usegdlibrary{trees}
\end{codeexample}
--]]
```

Reply:
done.